Foundation/NSFileManager.swift

```diff
@@ -9,10 +9,16 @@
 
 #if os(OSX) || os(iOS)
     import Darwin
-#elseif os(Linux)
+#elseif os(Linux) || os(Android)
     import Glibc
 #endif
 
+#if os(Android)
```

Comment:
I'd really like any special case for other platforms to at least come with a comment explaining why it's needed and what could be done to elide it in the future. Even better would be to avoid as much platform-specific code as possible, or put it into a central location so we can count on its presence everywhere in the project.